docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -4,6 +4,8 @@
 Cause Model Template
 ====================
 
+.. _Pull Request 64: https://github.com/ihmeuw/vivarium_research/pull/64
```

Comment:
Why are you holding on to links to PRs?

Reply:
To refer back to people's comments so we can see what decisions were made and why